kaspresso/src/main/kotlin/com/kaspersky/kaspresso/runner/KaspressoRunner.kt

```diff
@@ -0,0 +1,24 @@
+package com.kaspersky.kaspresso.runner
+
+import android.os.Bundle
+import androidx.test.runner.AndroidJUnitRunner
+import com.kaspersky.kaspresso.runner.listener.KaspressoRunNotifier
+import com.kaspersky.kaspresso.runner.listener.KaspressoRunNotifierImpl
+import com.kaspersky.kaspresso.runner.listener.SpyRunListener
+
+@Suppress("UNUSED")
+class KaspressoRunner : AndroidJUnitRunner(), KaspressoRunNotifier by KaspressoRunNotifierImpl() {
+
+    override fun onCreate(arguments: Bundle) {
+        arguments.putArgs("listener", SpyRunListener::class.java.name)
+        super.onCreate(arguments)
+    }
+
+    private fun Bundle.putArgs(key: String, vararg values: CharSequence?) {
+        val valuesArg: String = listOfNotNull<CharSequence>(
```

Comment:
The listeners class names are stored in a string, so we just add our listener to the end of it.